packages/fuselage-ui-kit/src/surfaces/FuselageSurfaceRenderer.tsx

```diff
@@ -528,4 +530,42 @@ export abstract class FuselageSurfaceRenderer extends UiKit.SurfaceRenderer<Reac
       </AppIdProvider>
     );
   }
+
+  channels_select(
+    block: UiKit.ChannelsSelectElement,
+    context: UiKit.BlockContext,
+    index: number
+  ): ReactElement | null {
+    if (context === UiKit.BlockContext.FORM) {
+      return (
+        <ChannelsSelectElement
+          block={block}
+          context={context}
+          index={index}
+          surfaceRenderer={this}
```

Comment:
More of a question, but, I see these values are passed to the element, but not used in them. Is there a reason for this?

Reply:
To be honest, I have the same question as you... I was following the pattern of the type definition itself. If you look at the type definition, there is a necessity to pass them in BlockProps... in some cases like in ActionBlocks surfaceRenderer is used... maybe this is some of the things we need to revalidate in a refactor.